c2h/include/c2h/half.cuh

```diff
@@ -324,28 +329,33 @@ inline std::ostream& operator<<(std::ostream& out, const __half& x)
 }
 
 /******************************************************************************
- * Traits overloads
+ * limits
  ******************************************************************************/
 
+_LIBCUDACXX_BEGIN_NAMESPACE_STD
 template <>
-struct CUB_NS_QUALIFIER::FpLimits<half_t>
+class __numeric_limits_impl<half_t, __numeric_limits_type::__floating_point>
 {
-  static __host__ __device__ __forceinline__ half_t Max()
+public:
+  static __host__ __device__ __forceinline__ half_t max()
   {
     return (half_t::max)();
   }
 
-  static __host__ __device__ __forceinline__ half_t Lowest()
+  static __host__ __device__ __forceinline__ half_t lowest()
   {
     return half_t::lowest();
   }
 };
+_LIBCUDACXX_END_NAMESPACE_STD
```

Comment:
@miscco Is this an acceptable way to specialize `cuda::std::numeric_limits` for a custom type? I initially specialized the trait directly, but got numeric errors in the tests, because `half_t` was CV-qualified somewhere and it did not pick up the specialization here.

Reply:
For internal use yes, but it should also work to directly specialize it

Reply:
Right, but then I would need to specialize it twice, for `half_t` and `const half_t`.